packages/react-native-web/src/exports/ImageBackground/index.js

```diff
@@ -16,12 +16,12 @@ import Image from '../Image';
 import StyleSheet from '../StyleSheet';
 import View from '../View';
 
-type ImageBackgroundProps = {
+type ImageBackgroundProps = {|
   ...ImageProps,
   imageRef?: any,
   imageStyle?: $PropertyType<ImageProps, 'style'>,
   style?: $PropertyType<ViewProps, 'style'>
-};
+|};
```

Comment:
I had `Flow` errors after my changes
Following the error messages and suggested fixes led to these changes